web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -152,11 +152,17 @@
          */
         createMap: function(type) {
           var config = gnMap.getMapConfig()['map-' + type];
+          
+          var proj = gnMap.getMapConfig().projection;
+          if (type === this.VIEWER_MAP) {
+            gnMap.checkProj4Def(proj);
+          }
+
```

Comment:
Whenever the viewer map is created, check if the Proj4 definition string is "standard" or custom.